README.md

```diff
@@ -164,4 +164,28 @@ git pull otax/master
 npm run compile
 pm2 restart prod-server
 ```
--->
\ No newline at end of file
+-->
+
+## Deployment
+
+### Heroku
+
+You can deploy Ilmomasiina to Heroku with the following steps: (note that you need a PostgreSQL database to connect to)
```

Comment:
Ilmomasiina fully supports MySQL (actually, it currently doesn't work fully with Postgres due to being developed with MySQL). We also already detect & parse the `CLEARDB_DATABASE_URL` env var for Heroku's shared ClearDB MySQL. This should be the default guidance for Heroku.

One can also get free and paid Postgres, and paid MySQL from Heroku, [That is configured into `DATABASE_URL`](https://devcenter.heroku.com/articles/heroku-postgresql#provisioning-heroku-postgres) which we could also support if we want.